black_it/samplers/gaussian_process.py

```diff
@@ -67,19 +66,20 @@ def __init__(  # pylint: disable=too-many-arguments
             random_state: the random state of the sampler, fixing this number the sampler behaves deterministically
             max_deduplication_passes: the maximum number of deduplication passes that are made
             candidate_pool_size: number of randomly sampled points on which the random forest predictions are evaluated
-            max_iters: maximum number of iteration in the optimization of the GP hyperparameters
             optimize_restarts: number of independent random trials of the optimization of the GP hyperparameters
             acquisition: type of acquisition function, it can be 'expected_improvement' of simply 'mean'
+            jitter: positive value to make the "expected_improvement" acquisition more explorative.
         """
         self._validate_acquisition(acquisition)
 
         super().__init__(
             batch_size, random_state, max_deduplication_passes, candidate_pool_size
         )
-        self.max_iters = max_iters
         self.optimize_restarts = optimize_restarts
         self.acquisition = acquisition
-        self._gpmodel: Optional[GPRegression] = None
+        self.jitter = jitter
```

Comment:
`optimize_restarts`, `acquisition` and `jitter` should be protected attributes. We can fix this in another PR.